### PR TITLE
Add JDDO feature.

### PR DESCRIPTION
**Pull request by daveoh**
### Change description ###
Add JDDO feature.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: cmc/values.yaml [hunk 1]
```diff
@@ -19,6 +19,8 @@ global:
   ccdCaseDataApiUrl: ""
   togglesUrl: ""
   postgresqlHost: ""
+  jddoEnabled: true
+  laDirectionOrderTemplate: "CV-CMC-GOR-ENG-0004.docx"
```
Comment (sabahirfan) [inline cmc/values.yaml]: Please move this back to application.yaml where other relevant fields belong. As this is going to be constant for all envs.
```
doc_assembly:
  url: ${DOC_ASSEMBLY_URL:false}
  templateId: "CV-CMC-GOR-ENG-0004.docx"
  judgeTemplateId: "CV-CMC-GOR-ENG-00219.docx"
<put it here pelase>
```

File: cmc/values.yaml [hunk 2]
```diff
@@ -66,6 +69,7 @@ cmc-claim-store:
       SEND_LETTER_URL: '{{ .Values.global.sendLetterUrl }}'
       DOCUMENT_MANAGEMENT_URL: '{{ .Values.global.documentManagementUrl }}'
       DOC_ASSEMBLY_URL: '{{ .Values.global.dgAssemblyUrl }}'
+      DOC_ASSEMBLY_TEMPLATEID: '{{ .Values.global.laDirectionOrderTemplate }}'
```
Comment (sabahirfan) [inline cmc/values.yaml]: This seems unnecessary here

File: cmc/values.yaml (hunk 1 above)
Comment (daveoh) [reply cmc/values.yaml]: It is not consistent. There are breaking changes so the templates are now versioned and need to be changed on a per environment basis

**Comment by klawani** [comment]
No need to touch this repository to get JDDO on HMCTS DEMO.  
When ready, let me know what config changes are required and I will take care of them.

**Pull request closed without merging**